packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -144,7 +203,7 @@ export default function MongoNav({
     `${cloudHost}/v2#/${projectId}`;
   const constructProjectURL = constructProjectURLProp ?? defaultProjectURL;
 
-  const constructedUrls: Required<URLSInterface> = {
+  const sanitizedURLs: Required<URLSInterface> = {
```

Comment:
"sanitized" to me implies something different than what you mean here.

Reply:
we were "sanitizing" the hosts so I thought it was a good parallel, can change back for sure if you want

Reply:
Sanitization usually implies removal of unsafe characters and information, e.g. to prevent script injection attacks.

Reply:
👍